springfield/cms/templates/cms/includes/wnp-footer.html

```diff
@@ -38,7 +38,9 @@
           </li>
         </ul>
       </nav>
-      <div class="fl-footer-logo" aria-hidden="true"></div>
+      <div class="fl-footer-logo" aria-hidden>
```

Comment:
```suggestion
      <div class="fl-footer-logo">
```

This made more sense when the logo was not a link. We should not hide this new functionality from users.